.github/workflows/pull_requests.yml

```diff
@@ -37,17 +37,11 @@ jobs:
     - name: Production Image Build
       if: success()
       id: build-image
-      uses: redhat-actions/buildah-build@v2
+      uses: docker/build-push-action@v6
       with:
         # Not being uploaded to any registry, use a simple name to allow Buildah to build correctly.
         image: cyberchef
-        containerfiles: ./Dockerfile
         platforms: linux/amd64
-        oci: true
-        # Webpack seems to use a lot of open files, increase the max open file limit to accomodate.
-        extra-args: |
-          --ulimit nofile=10000
```

Comment:
I haven't seen this issue with the docker actions myself, so let's hope it's not still an issue.